include/multigrid_pclomp/multi_voxel_grid_covariance_omp_impl.hpp

```diff
@@ -157,28 +189,33 @@ template<typename PointT>
 int MultiVoxelGridCovariance<PointT>::radiusSearch(const PointT &point, double radius, std::vector<LeafConstPtr> &k_leaves, unsigned int max_nn) const {
   k_leaves.clear();
 
-  // Find neighbors within radius in the occupied voxel centroid cloud
+  // First, search from the grid kdtree to find candidate grids
```

Comment:
@SakodaShintaro This comment should be changed to 
`// Search from the kdtree to find neighbors of @point`

Reply:
Fixed in https://github.com/tier4/ndt_omp/pull/48/commits/0d9be0e96ce2bfcb74868015064fddf1bae4272b